bugsnag-plugin-android-exitinfo/src/test/java/com/bugsnag/android/ExitInfoCallbackTest.kt

```diff
@@ -108,4 +109,15 @@ internal class ExitInfoCallbackTest {
         verify(nativeEnhancer, times(0)).invoke(event, exitInfo1)
         verify(anrEventEnhancer, times(0)).invoke(event, exitInfo1)
     }
+
+    @Test
+    fun testUnknownExitReasonAndImportance() {
+        `when`(exitInfos.first().processStateSummary).thenReturn("1".toByteArray())
+        `when`(event.session?.id).thenReturn("1")
+        `when`(exitInfo1.reason).thenReturn(ApplicationExitInfo.REASON_UNKNOWN)
+        `when`(exitInfo1.importance).thenReturn(ActivityManager.RunningAppProcessInfo.REASON_UNKNOWN)
+        assertTrue(exitInfoCallback.onSend(event))
+        assertEquals("0", exitInfo1.reason.toString())
+        assertEquals("0", exitInfo1.importance.toString())
```

Comment:
These two asserts are testing Mockito, not the `exitInfoCallback` behaviour. They should rather be checking for the expected values in the `event` Metadata